File: BitirmeTezi/ClientApp/src/models/Stream.ts
import { Streamer } from "./Streamer"

export class Stream{
    id!: number
    url!: string
    category!: string    
    streamer!: Streamer
    subtitle!: string

    constructor(id: number, url: string, category: string, streamer: Streamer, subtitle: string) {
        this.id = id
        this.url = url
        this.category = category
        this.streamer = streamer
        this.subtitle = subtitle
    }
}

File: BitirmeTezi/Models/Stream.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace BitirmeTezi.Models
{
    public class Stream
    {       
        public int Id { get; set; }
        public string URL { get; set; }
        public string Category { get; set; }
        public int StreamerId { get; set; }
        public string Subtitle { get; set; }
    }
}
